SR2025/2025-03-05-competition-attendance-followup.md

```diff
@@ -0,0 +1,20 @@
+---
+to: SR2025 teams who haven't RSVP'd
+subject: "SR2025 attendance followup"
+---
+
+Hi,
+
+We're working hard getting [the competition event][competition-event] ready for you. To best support your team, we need to know roughly how large your team is and whether there are any special considerations we need to take for anyone who is coming.
+
+We understand you might not know the exact number of team members who will attend, so vague guesses are absolutely fine for now! You'll be able to edit your response until our plans are finalised. If you still want to make changes after that, please send us an email and we'll see what we can do.
+
+[Fill out your team details](https://forms.gle/TTWgDCKS2hWcEUSE8)
+
+As ever if your team would like more help & support please do let us know.
+Most teams make a lot of progress as the competition nears, as well as at the event itself, so don't worry if it seems your team still has a lot to do!
+
+Thanks,
+-- SR Competition Team
```

Comment:
Not sure if you're aware -- we concluded at the start of this year that external folks don't really care or know which sub-team within SR is emailing them and that we'd just sign off as the organisation rather than a sub-team. This has (mostly) lead to greater simplicity & consistency.

```suggestion
-- Student Robotics
```